MapboxCoreNavigation/RouteController.swift

```diff
@@ -1,5 +1,6 @@
 import Foundation
 import CoreLocation
+import MapboxCommon
```

Comment:
> It doesn’t quite build yet due to a missing type declaration in MapboxNavigationNative.

This import fixed the build. Apparently MapboxCommon defines the MBXExpected class that MapboxNavigationNative now uses (but only forward-declares).